app/main/views/performance.py

```diff
@@ -12,8 +12,8 @@
 @main.route("/performance")
 def performance():
     stats = performance_dashboard_api_client.get_performance_dashboard_stats(
-        start_date=(datetime.utcnow() - timedelta(days=7)).date(),
-        end_date=datetime.utcnow().date(),
+        start_date=(datetime.now(current_app.config['PY_TIMEZONE']) - timedelta(days=7)).date(),
```

Comment:
[question] can this logic be used above where there's a `TODO` to get a time delta in local timezone?

Reply:
Yes! Thanks for finding my `TODO`